### ISSUE TITLE
Testrpc responds to web3.version.getNetwork with an increasing number

### ISSUE THREAD

**Issue by johnmcdowall** [ethereumjs/testrpc#189]
Testrpc responds to web3.version.getNetwork with an increasing number (the time when the testrpc was started). 

Although this prevents people from thinking they are running against mainnet etc, it's actually not useful to have it be non-deterministic and a different value on every startup. 

It's also surprising since real clients return `1` for mainnet etc. 

## Expected Behavior
Calls to `web3.version.getNetwork` should always return a value that is reserved to indicate that it is `testrpc` running.

## Current Behavior
Calls to `web3.version.getNetwork` return a timestamp set when `testrpc` starts up. 

## Possible Solution
Chose a particular value to represent a `testrpc` network, or prefix the timestamp with some agreed upon value like `888`.

## Steps to Reproduce (for bugs)
1. `window.web3.version.getNetwork(function(err, d) { console.log(d) })`

**Comment by axic** [comment]
Relevant: https://github.com/ethereum/interfaces/issues/6

**Comment by benjamincburns** [comment]
Agreed that a recognizable network ID would be ideal as a default.  In the mean time you can specify your own fixed network id using the `-i` or `--networkId` command line arguments.

**Comment by benjamincburns** [comment]
Open issue is whether the CLI and the UI should have the same network ID, as it might make sense to deconflict these...
